internal/api/index.go

```diff
@@ -16,6 +16,29 @@ import (
 	"github.com/photoprism/photoprism/pkg/txt"
 )
 
+// GET /api/v1/index
+func IndexingStatus(router *gin.RouterGroup) {
+	router.GET("/index", func(c *gin.Context) {
+		s := Auth(SessionID(c), acl.ResourcePhotos, acl.ActionRead)
+
+		if s.Invalid() {
+			AbortUnauthorized(c)
+			return
+		}
+
+		conf := service.Config()
+
+		if !conf.Settings().Features.Library {
+			AbortFeatureDisabled(c)
+			return
+		}
+
+		ind := service.Index()
+
+		c.JSON(http.StatusOK, gin.H{"running": ind.IsRunning()})
```

Comment:
I'd change "running" to "busy" so that it matches the backend name. True will also be returned when the importer is running, which blocks the main indexer as well.